src/styles/GlobalStyle.js

```diff
@@ -4,12 +4,13 @@ export default createGlobalStyle`
 
 :root {
     
-
+    --warning-red: #de5246;
     --almost-black: #212121; //font-color
```

Comment:
Would probably change the naming, in order to know, what the color is for and not to have to write comments. For example: ``--font-color: #212121``